### PR TITLE
Fix misspelled words, Improve chat title bar

### PR DESCRIPTION
**Pull request by Muaath5**
**What new features or bug fixes does this pull request introduce? Explain.**
- Chat title bar
- Fix misspelled file names & icons

**Is this pull request related to an existing issue?**
Probably

**Does this pull request comply with the conditions described in contribution [guidelines](https://github.com/DIBgram/DIBgram/blob/main/CONTRIBUTING.md#submitting-a-pull-request)?**

- [x] There are no errors, warnings or notices from ESLint or StyleLint
- [x] It compiles and runs successfully
- [x] It does not introduce a feature that is not present in the official Telegram Desktop app
- [x] Behavior and UI is similar to the official Telegram Desktop app

### CODE REVIEW

File: src/dibgram/messenger/sections/chat/headers/title.scss [hunk 1]
```diff
@@ -16,5 +16,20 @@
...
             color: var(--theme-color-dialogsNameFg);
         }
+        .info {
+            font-size: 14px;
+            font-weight: 400;
+            color: var(--theme-color-windowSubTextFg);
+        }
+    }
+    .title-bar-right {
+        flex-grow: 1;
```
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.scss:26]: Why `flex-grow: 1;`?

File: src/dibgram/messenger/sections/chat/headers/title.scss [hunk 2]
```diff
@@ -16,5 +16,20 @@
...
+        padding-inline-end: 15px;
+
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        align-items: flex-end;
```
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.scss:33]: Are you sure a flexbox is needed here?

File: src/dibgram/messenger/sections/chat/headers/title.js [hunk 3]
```diff
@@ -9,6 +12,17 @@ export default function TitleHeader({chat}) {
                 <div className="title">
                     {chat.title}
                 </div>
+                <div className="info">
+                    {(chat.type['@type'] == 'chatTypeSupergroup' && chat.type.is_channel) && __pl('lng_chat_status_subscribers', 5)}
+                    {(chat.type['@type'] == 'chatTypeBasicGroup' || (chat.type['@type'] == 'chatTypeSupergroup' && !chat.type.is_channel)) && __pl('lng_chat_status_members', 5)}
```
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.js:17]: Not all groups or channels have 5 members.

File: src/dibgram/messenger/sections/chat/headers/title.js [hunk 4]
```diff
@@ -9,6 +12,17 @@ export default function TitleHeader({chat}) {
...
+                </div>
+            </div>
+            <div className="title-bar-right">
+                <ThreeDotsMenu>
+                    <Menu.MenuContents>
+                        <Menu.MenuItem>{chat.can_be_reported && __('lng_report_button')}</Menu.MenuItem>
```
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.js:23]: The condition should be outside the menu item.

**Review by Mammad900 — CHANGES_REQUESTED** [review]
I've found some errors in your pull request. Please check them.

File: src/dibgram/messenger/sections/chat/headers/title.scss (hunk 2 above)
Comment (Muaath5) [reply src/dibgram/messenger/sections/chat/headers/title.scss:33]: I don't know how to place it in the right place unless this way.

**Comment by Mammad900** [comment]
```
Failed to compile.

./src/dibgram/messenger/sections/settings/choose-lanugage.js
Error: ENOENT: no such file or directory, open '[redacted]\src\dibgram\messenger\sections\settings\choose-lanugage.js'  
```
You renamed the file but did not fix the imports

File: src/dibgram/messenger/sections/chat/headers/title.js (hunk 4 above)
Comment (Mammad900) [reply src/dibgram/messenger/sections/chat/headers/title.js:23]: image
This is what happens when a chat isn't reportable

File: src/dibgram/messenger/sections/chat/headers/title.scss (hunk 2 above)
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.scss:33]:
```suggestion
        flex-shrink: 0;

        & > .three-dots-menu,
        & > .icon-button {
            float: unset;
            margin: 7px;
        }
```

File: src/dibgram/messenger/sections/chat/headers/title.js (hunk 4 above)
Comment (Mammad900) [reply src/dibgram/messenger/sections/chat/headers/title.js:23]:
```suggestion
                        {chat.can_be_reported && <Menu.MenuItem>{__('lng_report_button')}</Menu.MenuItem>}
```

**Comment by Muaath5** [comment]
@Mammad900 I forgot to do Ctrl+S XD

**Comment by Mammad900** [comment]
@Muaath5 Don't worry, I fixed it for you.

File: src/dibgram/messenger/sections/chat/headers/title.scss [hunk 5]
```diff
@@ -16,5 +16,21 @@
             font-weight: 500;
             color: var(--theme-color-dialogsNameFg);
         }
+        .info {
+            font-size: 14px;
+            font-weight: 400;
+            color: var(--theme-color-windowSubTextFg);
+        }
```
Comment (Mammad900) [inline src/dibgram/messenger/sections/chat/headers/title.scss:23]:
```suggestion
        .info {
            margin-top: 2px;
            font-size: 13px;
            font-weight: 400;
            color: var(--theme-color-windowSubTextFg);
        }
```

**Review by Mammad900 — APPROVED** [review]

**Pull request merged**